app/scripts/modules/core/src/application/nav/ApplicationNavigation.spec.tsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { mock } from 'angular';
+import { UIRouterReact, UIRouterContext } from '@uirouter/react';
+import { StateMatcher } from '@uirouter/core';
+
+import { REACT_MODULE } from '../../reactShims';
+import { OVERRIDE_REGISTRY } from '../../overrideRegistry';
+import {
+  mockAppConfigDataSourceConfig,
+  mockServerGroupDataSourceConfig,
+  mockLoadBalancerDataSourceConfig,
+  mockTaskDataSourceConfig,
+  mockPipelineDataSourceConfig,
+} from '@spinnaker/mocks';
+import { ApplicationModelBuilder } from '../../application';
+import { IPipeline } from '../../domain';
+import { ApplicationDataSource } from '../service/applicationDataSource';
+
+import { ApplicationNavigation } from './ApplicationNavigation';
+
+describe('ApplicationNavigation', () => {
+  let $uiRouter: UIRouterReact;
+  const currentStates = ['**.pipelines.**', '**.tasks.**'];
+
+  beforeEach(mock.module(REACT_MODULE, OVERRIDE_REGISTRY));
+  beforeEach(
+    mock.inject((_$uiRouter_: UIRouterReact) => {
+      $uiRouter = _$uiRouter_;
+      $uiRouter.globals.current.name = '**.tasks.**';
```

Comment:
this sort of thing still seems unnecessary to me, but since it's not hurting anything let's just move on

Reply:
I don't mind deleting it. In this case, the `uirouter` looks at this value for the initial route, rather than having that be defined.